package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "anumargak",
-  "version": "1.4.1",
+  "version": "1.5.0",
```

Comment:
Please revert this change. I prefer to update the package related information just before the release.

Reply:
Hmm! I can see. Git is showing my many changes as a part of this PR. Ignore it for the time being. Once all the changes are done, we'll squash the changes. I hope we should not see any issue that time.